cfn-templates/data-exports-aggregation.yaml

```diff
@@ -0,0 +1,749 @@
+              # SSEAlgorithm: aws:kms
+              # KMSMasterKeyID: "REPLACE_WITH_YOUR_KEY_ARN"
+      AccessControl: BucketOwnerFullControl
+      OwnershipControls:
+        Rules:
+          - ObjectOwnership: BucketOwnerEnforced
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      VersioningConfiguration:
+        Status: Enabled
+      ReplicationConfiguration:
+        Role: !GetAtt CrossRegionReplicationRole.Arn
+        Rules:
+          -
+            Destination:
+              Bucket:
+                Fn::Sub: "arn:${AWS::Partition}:s3:::${ResourcePrefix}-${DestinationAccountId}-shared"
+              StorageClass: STANDARD
+            Id: ReplicationRule1
+            Prefix: ""
+            Status: Enabled
+      LifecycleConfiguration:
+        Rules:
+          - Id: Object&Version Expiration
+            Status: Enabled
+            NoncurrentVersionExpirationInDays: 32 # 1 month
+            ExpirationInDays: 64 # 2 months
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: 'W35'
+            reason: "Data buckets would generate too much logs"
+      cfn-lint:
+        config:
+          ignore_checks:
+            - W3045 # Need to use AccessControl for replication
+
+  SourceS3BucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Condition: IsSourceAccount
+    DeletionPolicy: Delete
+    UpdateReplacePolicy: Delete
+    Properties:
+      Bucket: !Ref SourceS3
+      PolicyDocument:
+        Id: CrossAccessPolicy
+        Version: "2012-10-17"
+        Statement:
+          - Sid: AllowTLS12Only
+            Effect: Deny
+            Principal: "*"
+            Action: s3:*
+            Resource:
+              - !Sub 'arn:${AWS::Partition}:s3:::${SourceS3}'
+              - !Sub 'arn:${AWS::Partition}:s3:::${SourceS3}/*'
+            Condition:
+              NumericLessThan:
+                s3:TlsVersion: 1.2
+          - Sid: AllowOnlyHTTPS
+            Effect: Deny
+            Principal: "*"
+            Action: s3:*
+            Resource:
+              - !Sub 'arn:${AWS::Partition}:s3:::${SourceS3}'
+              - !Sub 'arn:${AWS::Partition}:s3:::${SourceS3}/*'
+            Condition:
+              Bool:
+                aws:SecureTransport: false
+          - Sid: AllowReadBilling
+            Effect: Allow
+            Principal:
+              Service: 
+                - billingreports.amazonaws.com
+                - bcm-data-exports.amazonaws.com
+            Action:
+              - s3:GetBucketAcl
+              - s3:GetBucketPolicy
+            Resource:
+              - !Sub 'arn:${AWS::Partition}:s3:::${SourceS3}'
+              - !Sub 'arn:${AWS::Partition}:s3:::${SourceS3}/*'
+          - Sid: AllowWriteBilling
+            Effect: Allow
+            Principal:
+              Service: 
+                - billingreports.amazonaws.com
```

Comment:
is this still needed for CUR2?

Reply:
`billingreports.amazonaws.com` is not needed for Data Exports/CUR2 but we've been adding it in our UI workflow in case customers are trying to deliver Legacy CURs to the same bucket.

We could remove it here if we think the bucket would only be used for this.

Reply:
I think mixing CUR legacy and CUR 2 in the same bucket can cause a lot of issues with Athena and Glue. I don't think it worth to consider this use case for CID template

Reply:
Ah makes sense - okay will remove